products/compute/terraform.yaml

```diff
@@ -236,6 +242,10 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         is_set: true
       cdnPolicy.cacheKeyPolicy.queryStringBlacklist: !ruby/object:Overrides::Terraform::PropertyOverride
         is_set: true
+      cdnPolicy.cacheMode: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
```

Comment:
Question to inform me in making future resources: It doesn't look like there's anything in the docs specifying this; do you have a method for figuring this out apart from experimentation?

Reply:
Sadly no!